src/bin/stratis_dumpmetadata.rs

```diff
@@ -0,0 +1,53 @@
+use std::{
+    env,
+    fs::OpenOptions,
+    io::{Seek, SeekFrom},
+    process,
+    vec::Vec,
+};
+
+use serde_json::Value;
+
+use libstratis::engine::BDA;
+
+fn run(devpath: String) -> Result<(), String> {
```

Comment:
Step (1): Please change parameter type to &str. We don't need to copy the string, we just needed a reference to it.

Reply:
Resolved.